machine/v1beta1/types_awsprovider.go

```diff
@@ -107,6 +107,22 @@ type AWSMachineProviderConfig struct {
 	// If this value is selected, capacityReservationID must be specified to identify the target reservation.
 	// +optional
 	MarketType MarketType `json:"marketType,omitempty"`
+
+	// hostID specifies the Dedicated Host on which the instance must be started.
+	// This field is mutually exclusive with DynamicHostAllocation.
+	// +kubebuilder:validation:Pattern=`^h-[0-9a-f]{17}$`
+	// +kubebuilder:validation:MaxLength=19
+	// +openshift:enable:FeatureGate=AWSDedicatedHosts
+	// +optional
```

Comment:
Please include constraints, explained in plain sentences, in the GoDoc.

Reply:
I'll update this.